rs-client-demo/src/main.rs

```diff
@@ -29,133 +29,131 @@ fn main() {
+            ("main-preview","search-default-override-allowlist"),
+            ("main","cfr-fxa"),
+            ("main-preview","cfr-fxa"),
+            ("main", "personality-provider-recipe"),
+            ("main-preview", "personality-provider-recipe"),
+            ("main", "personality-provider-models"),
+            ("main-preview", "personality-provider-models"),
+            ("main", "cfr-ml-control"),
+            ("main-preview", "cfr-ml-control"),
+            ("main", "cfr-ml-experiments"),
+            ("main-preview", "cfr-ml-experiments"),
+            ("main", "cfr-ml-models"),
+            ("main", "public-suffix-list"),
+            ("blocklists-preview", "qa"),
+            ("main", "hijack-blocklists"),
+            ("main-preview", "hijack-blocklists"),
+            ("main", "cfr-srg"),
+            ("main-preview", "cfr-srg"),
+            ("main", "language-dictionaries"),
+            ("main-preview", "language-dictionaries"),
+            ("main", "fftv-experiments"),
+            ("main-preview", "fftv-experiments"),
+            ("main", "tracking-protection-lists"),
+            ("main-preview", "tracking-protection-lists"),
+            ("main", "anti-tracking-url-decoration"),
+            ("main-preview", "anti-tracking-url-decoration"),
+            ("main", "tippytop"),
+            ("main-preview", "tippytop"),
+            ("main", "focus-experiments"),
+            ("main-preview", "focus-experiments"),
+            ("main", "fenix-experiments"),
+            ("main-preview", "fenix-experiments"),
+            ("main", "lite-experiments"),
+            ("main-preview", "lite-experiments"),
+            ("main", "sites-classification"),
+            ("main-preview", "sites-classification"),
+            ("main", "rocket-prefs"),
+            ("main-preview", "rocket-prefs"),
+            ("blocklists-preview", "gfx"),
+            ("blocklists", "qa"),
+            ("pinning-preview", "pins"),
+            ("pinning", "pins"),
+            ("blocklists", "gfx")
+        ];
+
+    let mut failed_fetch = Vec::new();
+
+    for collection in &collections {
+        println!(" -- --- -- -- -- -- - -");
+        println!("{}, {}", collection.0, collection.1);
         
-    //     let client = Client::create_with_bucket_collection(collection.0, collection.1, None);
+        let client = Client::create_with_bucket_collection(collection.0, collection.1, None);
 
-    //     println!("Fetching records");
+        println!("Fetching records");
 
-    //    match client.get(0) {
-    //        Ok(records) => println!("Successful fetch"),
-    //        Err(error) => {
-    //            println!("Could not fetch records: {:?}", error);
-    //            failed_fetch.push(collection);
-    //        },
-    //    };
+       match client.get() {
+           Ok(records) => println!("Successful fetch"),
+           Err(error) => {
+               println!("Could not fetch records: {:?}", error);
```

Comment:
It's not only about fetching. Signature verification can fail, right?

Reply:
yes

Reply:
Then we should adjust the message ;)

Reply:
agreed